### PR TITLE
r_torch_tutorial

### PR DESCRIPTION
**Pull request by Zhao-Wenbo**
Wenbo Zhao, wz2554

### CODE REVIEW

File: r_torch_tutorial.Rmd [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    self$conv1 <- nn_conv2d(1, 32, 3, 1)
+    self$conv2 <- nn_conv2d(32, 64, 3, 1)
+    self$dropout1 <- nn_dropout2d(0.25)
+    self$dropout2 <- nn_dropout2d(0.5)
+    self$fc1 <- nn_linear(9216, 128)
+    self$fc2 <- nn_linear(128, 10)
+  },
+  
+  forward = function(x) {
+    x %>%                                  # N * 1 * 28 * 28
```
Comment (rao53) [inline r_torch_tutorial.Rmd:103]: There should be no additional single hashtag headers in the chapter. (If there are, new chapters will be created.)

**Comment by rao53** [comment]
Thank you for your contribution!

**Comment by Zhao-Wenbo** [comment]
Thank you very much!

**Comment by rao53** [comment]
@jtr13 I'm not sure why the job workflow is failing.. Is it because of the previous workflows failing?

**Comment by jtr13** [comment]
Thanks for letting me know... I just realized that it's been failing for a while. I'm working on fixing it.

**Pull request merged**